.github/workflows/inspektor-gadget.yml

```diff
@@ -451,6 +464,67 @@ jobs:
 
         perl tools/check-readme.pl ./kubectl-gadget README.md
 
+  test-integration-aks:
+    name: Integration tests on AKS
+    # level: 1
+    needs: [check-secrets, test-unit, build-kubectl-gadget, build-local-gadget, build-gadget-container-images, publish-gadget-images-manifest]
+    if: needs.check-secrets.outputs.aks == 'true'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os-sku: [Ubuntu]
+        arch: [amd64, arm64]
+    steps:
+    - uses: azure/login@v1
+      name: Login to Azure
+      with:
+        creds: ${{ secrets.AZURE_AKS_CREDS }}
+    - name: Craft cluster name
+      shell: bash
+      run: |
+        cluster_name=${{ env.AZURE_AKS_CLUSTER_PREFIX }}${{ matrix.arch }}-${{ matrix.os-sku }}
+
+        echo "CLUSTER_NAME=${$cluster_name}" >> GITHUB_ENV
+    - name: Create AKS cluster ${{ env.CLUSTER_NAME }}
+      shell: bash
+      run: |
+        cluster_name=${{ env.AZURE_AKS_CLUSTER_PREFIX }}${{ matrix.arch }}-${{ matrix.os-sku }}
+
+        node_size='Standard_D2s_v5'
+        if [ ${{ matrix.arch }} = 'arm64' ]; then
+          # 'p' means the node size corresponds to arm64 hardware.
+          node_size='Standard_D2ps_v5'
+        fi
+
+        # Let's keep thing in the US to avoid data crossing the Atlantic as
+        # GitHub data centers are in the US:
+        # https://github.blog/2017-10-12-evolution-of-our-data-centers/
+        az aks create -l westcentralus -g ${{ secrets.AZURE_AKS_RESOURCE_GROUP }} -n ${{ env.CLUSTER_NAME }} -s $node_size --os-sku ${{ matrix.os-sku }} --no-ssh-key
+    - uses: azure/aks-set-context@v3
+      name: Set AKS cluster ${{ env.CLUSTER_NAME }} context
+      with:
+        cluster-name: ${{ env.CLUSTER_NAME }}
+        resource-group: ${{ secrets.AZURE_AKS_RESOURCE_GROUP }}
+        admin: false
+    - uses: actions/checkout@v3
+      with:
+        ref: 'main'
+    - name: Run integration tests
+      uses: ./.github/actions/run-integration-tests
+      env:
+        KUBERNETES_DISTRIBUTION: "aks-${{ matrix.os-sku }}"
+      with:
+        registry: ${{ env.REGISTRY }}
+        container_repo: ${{ env.CONTAINER_REPO }}
+        image_tag: 'latest'
```

Comment:
Shouldn't you use `./.github/actions/set-container-repo-and-determine-image-tag` to get this info?